Content.Client/UserInterface/RoundEndSummaryWindow.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Robust.Client.Graphics;
+using Robust.Client.Interfaces.Input;
+using Robust.Client.Interfaces.ResourceManagement;
+using Robust.Client.ResourceManagement;
+using Robust.Client.UserInterface.Controls;
+using Robust.Client.UserInterface.CustomControls;
+using Robust.Shared.IoC;
+using Robust.Shared.Localization;
+using Robust.Shared.Maths;
+using Content.Client.Utility;
+
+namespace Content.Client.UserInterface
+{
+    public sealed class RoundEndSummaryWindow : SS14Window
+    {
+        private readonly int _headerFontSize = 14;
+        private VBoxContainer VBox { get; }
+
+        protected override Vector2? CustomSize => (520, 580);
+
+#pragma warning disable 649
+        [Dependency] private readonly IResourceCache _resourceCache;
+        [Dependency] private readonly ILocalizationManager _loc;
+        [Dependency] private readonly IInputManager _inputManager;
+#pragma warning restore 649
+
+        public RoundEndSummaryWindow(string gm, uint duration)
+        {
+            Title = "Round End Summary";
+
+            //Get section header font
+            _loc = IoCManager.Resolve<ILocalizationManager>();
```

Comment:
You can access the localization manager statically via `Robust.Shared.Localization.Loc`. Don't have to resolve it.